gno.land/pkg/gnoweb/frontend/js/controller-action-function.ts

```diff
@@ -106,17 +106,28 @@ export class ActionFunctionController extends BaseController {
 				arg.textContent = escapedValue || "";
 			});
 
-		// Update function link with new parameter value
-		const functionLink = this.getTarget("function-link") as HTMLAnchorElement;
-		if (functionLink) {
-			const currentUrl = functionLink.getAttribute("href");
-			if (!currentUrl) {
-				console.warn(`No href attribute found for function link`);
-				return;
-			}
-			const u = new URL(functionLink.href, window.location.origin);
-			u.searchParams.set(paramName, paramValue);
```

Comment:
Previously, this would append the Execute link with something like `?username=testuser` where the param has the value that is dynamically typed by the user. Now this is gone from the Execute link.

Reply:
No both are dynamic... 

https://github.com/user-attachments/assets/249364b6-5705-4adb-ab33-0b5416582d78

Reply:
I see it now on my local testing. Sorry for the confusion.

Another question: When I click "Click here to register" on https://gno.land/r/gnoland/users/v1, it opens the help page with https://gno.land/r/gnoland/users/v1$help&func=Register&.send=1000000ugnot . Is it possible to preserve `.send=1000000ugnot` in the Link and Execute URLs?